website/install.md

```diff
@@ -72,3 +72,11 @@ Git Town runs anywhere Git and [Bash](https://www.gnu.org/software/bash/bash.htm
 #### Install autocompletion
 
 * for [Fish shell](http://fishshell.com): `git town install-fish-autocompletion`
+
+
+#### Notifications about new releases
+
+* Subscribe to our
+  <a href="https://github.com/Originate/git-town/releases.atom)
```

Comment:
The close paren needs to be removed and the part of the sentence to "never miss a new release!" needs to be moved to after the link